existing_layers/all_regions.txt

```diff
@@ -0,0 +1,27 @@
+ap-south-2
+ap-south-1
+eu-south-1
+eu-south-2
+me-central-1
+ca-central-1
+eu-central-1
+eu-central-2
+us-west-1
+us-west-2
+af-south-1
+eu-north-1
+eu-west-3
+eu-west-2
+eu-west-1
+ap-northeast-3
+ap-northeast-2
+me-south-1
+ap-northeast-1
+sa-east-1
+ap-east-1
+ap-southeast-1
+ap-southeast-2
+ap-southeast-3
+ap-southeast-4
+us-east-1
+us-east-2
```

Comment:
It would be nice to have an automated way to generate this, since we will be adding more in the future. It would be a pain to have to update this one file every time we add a new region.

Reply:
Yep this is done already in this function
https://github.com/DataDog/datadog-lambda-extension/pull/166/files#diff-260f0a0f31567ce6be2509b618b7298adfdb12ae2a72191a93f6b17d4cfb7827R1

I saved the file here since it doesn't happen very often